packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -38,7 +38,7 @@ contract Bridge is EssentialContract, IBridge, BridgeErrors {
 
     event DestChainEnabled(uint256 indexed chainId, bool enabled);
 
-    /// Allow Bridge to receive ETH from the TaikoL1, TokenVault or EtherVault.
```

Comment:
Please search for "token_vault" and replace them with "erc20_vault". 
Also, in this file, you will find:

```solidity
 /// Allow Bridge to receive ETH from the TaikoL1, ERC20Vault or EtherVault.
    receive() external payable {
        if (
            msg.sender != resolve("token_vault", true)
                && msg.sender != resolve("ether_vault", true)
                && msg.sender != resolve("taiko", true) && msg.sender != owner()
        ) {
            revert B_CANNOT_RECEIVE();
        }
    }

```

You need to add "erc721_vault", "erc1155_vault" to the list.